source/uwp/AdaptiveCards.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Let's make sure that the changes to this file are not checked in.

Reply:
absolutely!

---
In reply to: [154430317](https://github.com/Microsoft/AdaptiveCards/pull/1003#discussion_r154430317) [](ancestors = 154430317)